src/main/java/core/basesyntax/supplier/FigureSupplier.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax.supplier;
+
+import core.basesyntax.color.Color;
+import core.basesyntax.figure.Circle;
+import core.basesyntax.figure.Figure;
+import core.basesyntax.figure.IsoscelesTrapezoid;
+import core.basesyntax.figure.Rectangle;
+import core.basesyntax.figure.RightTriangle;
+import core.basesyntax.figure.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int NUMBEROFFIGURES = 5;
+    private static final int DEFAULTCIRCLESIZE = 10;
+    private static final int MAXVALUE = 1000;
+    private Random random = new Random();
+    private ColorSupplier color = new ColorSupplier();
+
+    public Figure getDefaultFigure() {
+        return new Circle(DEFAULTCIRCLESIZE,Color.WHITE.name());
+    }
+
+    public Figure getRandomFigure() {
+        switch (random.nextInt(NUMBEROFFIGURES)) {
+            case 0:
+                return new Circle(random.nextInt(MAXVALUE), color.getRandomColor());

```

Comment:
No reason to write return in all cases, it is possible to define method level variable and populate it in cases, to avoid writing return